Directory.Packages.props

```diff
@@ -25,14 +25,12 @@
 
   <ItemGroup Label=".NET 6 Specific" Condition="'$(TargetFramework)' == 'net6.0'">
     <!-- RabbitMQ.Stream.Client -->
-    <PackageVersion Include="System.IO.Hashing" Version="6.0.0" />
```

Comment:
We have other dependencies in the `Tests.csproj`  like:
```
  <PackageReference Include="AltCover" />
    <PackageReference Include="AmqpNetLite" />
    <PackageReference Include="Microsoft.NET.Test.Sdk" />
    <PackageReference Include="RabbitMQ.Client" />
    <PackageReference Include="System.IO.Hashing" />
  ```
And we don't need to do that. I am sure there is a reason why `System.IO.Hashing` needs that.

Reply:
The `Directory.Packages.props` file centralizes the versions of _all_ of the dependencies in _all_ of the `*.csproj` files. Since `Tests.csproj` has a dependency on `System.IO.Hashing`, that library must be present in `Directory.Packages.props`